extensions/wakatime/src/today-summary.tsx

```diff
@@ -0,0 +1,23 @@
+import { useSummary } from "./hooks";
+
+import { MenuBarExtra, getPreferenceValues } from "@raycast/api";
+
+export default function Command() {
+  const summary = useSummary();
+  const preference = getPreferenceValues();
+  const asksSentence = preference.showAsksSentence;
+
+  function getTitle(): string {
+    return `${summary.data
+      ?.map(
+        ([key, range]) =>
+          key === "Today" &&
+          (range.cummulative_total.seconds === 0 && asksSentence
+            ? asksSentence
+            : `Today: ${range.cummulative_total.text}`)
+      )
+      .filter((item) => item)}`;
+  }
+
+  return <MenuBarExtra icon="../assets/icon.png" title={getTitle()} isLoading={summary.isLoading} />;
```

Comment:
With all my suggestions, I hope for it to look something like this. Where `null` is returned when the preference is disabled and the time for `Today` is 0.

```ts
const title = getTitle();

// Strictly false meaning today's time is 0
return title === false && [preferenceToShowZeroDays] !== true ? null : (
  <MenuBarExtra
    icon="../assets/icon.png"
    // isLoading expected to be true when title is undefined
    // and false when title is false or the time as a string
    isLoading={summary.isLoading}
    title={title || "Are we coding today?"}
  />
);

```